code/modules/reagents/reagent_containers/pill.dm

```diff
@@ -42,6 +42,14 @@
 		var/mob/living/carbon/human/H = M
 		if(H.species.chem_flags & NO_EAT)
 			src.forceMove(get_turf(H))
+			if(istype(H.species,/datum/species/plasmaman))
+				if(istype(H.head,/obj/item/clothing/head/helmet))
+					H.visible_message("<span class='warning'>\The [src] slides off of the biosuit helmet, and falls to the floor.</span>", "<span class='notice'>\The [src] bounces off of the biosuit helmet, and falls to the floor</span>")
+				return 0
+			else
+				H.visible_message("<span class='warning'>\The [src] slips through a hole within the putrid jaw and falls to the floor.</span>", "<span class='notice'>You watch as \the [src] falls to the ground below you.</span>")
+			return 0
+		else
```

Comment:
uhhhh, pretty sure you don't want that else there - can you test if humans can eat pills now

Reply:
will have to test that tonight after work.